### DISCUSSION TITLE
What happened to 'CefSettingsBase.PersistUserPreferences' property ???

### DISCUSSION THREAD

**Discussion by ElektroStudios** [cefsharp/CefSharp#4955] [Q&A] [answered]
I've updated the nuget package and this example code is not working anymore due the lack of `CefSettingsBase.PersistUserPreferences` property:

```
Dim settings As New CefSettings With {
    .PersistUserPreferences = False
}
```

I've checked the changelog and didn't found anything. And if I search for term "PersistUserPreferences" in the repository, it throws no results.

It was deleted from the `CefSettingsBase` class without explanation?.

How is it supposed to be done now, with the latest version of CefSharp, so that user preferences are not persisted?.

Thank you in advance.

**Comment by campersau** [comment, marked as answer]
It was removed with the removal of Alloy Bootstrap in https://github.com/cefsharp/CefSharp/commit/19739abd9cc69529ecd06e86400e6f1068f14e6c

Upstream:

> persist_user_preferences (will be removed; always enabled with Chrome bootstrap)

https://github.com/chromiumembedded/cef/issues/3685#issuecomment-2091178215

**Reply by ElektroStudios** [reply]
> It was removed with the removal of Alloy Bootstrap in [19739ab](https://github.com/cefsharp/CefSharp/commit/19739abd9cc69529ecd06e86400e6f1068f14e6c)
> 
> Upstream:
> 
> > persist_user_preferences (will be removed; always enabled with Chrome bootstrap)
> 
> [chromiumembedded/cef#3685 (comment)](https://github.com/chromiumembedded/cef/issues/3685#issuecomment-2091178215)

Thank you very much for the help and quick reply!

But, excuse my ignorance and inexperience with CEF, "always enabled with Chrome bootstrap" - does this mean there is no longer a way to opt out of them persisting?.

That is the last thing I would like to know before closing the topic. Thanks again.

**Reply by amaitland** [reply]
> does this mean there is no longer a way to opt out of them persisting

I'm not aware of a way to opt out. If you can find a way to configure `Chrome` then the same solution should work.
